server/routes/api/homes.js

```diff
@@ -32,49 +16,106 @@ router.get('/:homeId', (req, res) => {
       id = null;
     }
   } else if (Number.parseInt(req.params.homeId, 10) > 0
-    && Number.parseInt(req.params.homeId, 10) <= 10000001) {
+    && Number.parseInt(req.params.homeId, 10) <= 10000002) {
     id = Number.parseInt(req.params.homeId, 10);
   } else {
     id = null;
   }
-  // const randomId = createRandomId();
-  Home.findByPk(id).then((home) => {
-    if (!home) {
-      return res.status(404).json({ noHomeFound: 'No home found with that ID' });
-    }
-    const { zipCode } = home;
-    Home.findAll({ where: { zipCode }, exclude: [{ id }] }).then((homes) => {
-      if (!homes) {
-        return res.status(404).json({ noHomesFound: 'no homes found with that ZIP Code' });
-      }
-      return res.status(200).json(homes);
-    });
-    return false;
-  });
+  if (id !== null) {
+    const query = 'SELECT * FROM neighborhood.homes where home_id = ?';
+    return client.execute(query, [id], { prepare: true })
+      .then((result) => {
+        const singleHome = result.rows[0];
+        const query2 = 'SELECT * FROM neighborhood.homes WHERE zipcode = ?';
+        client.execute(query2, [singleHome.zipcode], { prepare: true })
+          .then((newResult) => {
+            const arr = [];
+            for (let i = 0; i < 15; i += 1) {
+              if (newResult.rows[i].home_id !== id) {
+                arr.push(newResult.rows[i]);
+              }
+            }
+            if (arr.length <= 1) {
+              return res.status(404).json({ noHomeFound: 'No home found with that ID' });
+            }
+            return res.status(200).json(arr);
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+  return res.status(404).json('No home found with that ID');
 });
 
 router.post('/', (req, res) => {
   const { body } = { body: req.body };
-  Home.create(body)
-    .then(() => {
-      res.status(200);
-    });
+  const query = `INSERT INTO neighborhood.homes(home_id, home_name, dateOfPosting, status, numberOfLikes, numberOfBathroom, numberOfBedroom, homeValue, sqft, streetName, cityName, stateName, zipCode, homeImage) VALUES (${body.home_id}, ${body.home_name}, ${body.dateOfPosting}, ${body.status}, ${body.numberOfLikes}, ${body.numberOfBathroom}, ${body.numberOfBedroom}, ${body.homeValue}, ${body.sqft}, ${body.streetName}, ${body.cityName}, ${body.stateName}, ${body.zipCode}, ${body.homeImage})`;
```

Comment:
although this is backend code, it's def reach the max suggested line. I would recommend creating an object and assigning the values and then inserting it as a JSON or if possible - doing some sort of destructuring? 

https://www.datastax.com/dev/blog/whats-new-in-cassandra-2-2-json-support
https://docs.datastax.com/en/cql/3.3/cql/cql_using/useInsertJSON.html
```suggestion
const data = {
home_id: body.home_id
home_name: body.home_name
dateOfPosting: body.dateOfPosting,
status: body.status,
numberOfLikes: body.numberOfLikes
...
}

const query = `INSERT INTO neighborhood.homes JSON ?`;
client.execute(query, [data],)....
```

Reply:
Possible, but since I'm not positive it'll work and it's something I'm not running often, I don't really want to rock the boat